src/web/server/jaia_portal.py

```diff
@@ -46,6 +46,49 @@ def protobufMessageToDict(message):
     return google.protobuf.json_format.MessageToDict(message, preserving_proto_field_name=True)
 
 
+def filterDuplicateTaskPackets(taskPackets: list[dict]):
```

Comment:
Tried to run in Sim and I got this Error:
Traceback (most recent call last):
  File "./app.py", line 22, in <module>
    import jaia_portal
  File "/home/ferrom/repos/jaiabot/src/web/server/jaia_portal.py", line 49, in <module>
    def filterDuplicateTaskPackets(taskPackets: list[dict]):
TypeError: 'type' object is not subscriptable

I have python version 3.8 and the bots do as well.

Reply:
I should probably stick with that old way of type hinting until the world updates to the later 3.x versions of python...